### PR TITLE
Stability Scope Format Fix

### PR DESCRIPTION
**Pull request by Hipogeo**
@blimpa @Gala-G

### CODE REVIEW

File: MIP104/MIP104.md [hunk 1]
```diff
@@ -210,71 +232,131 @@ AllocatorDAO Advisory teams are subject to requirements for how they deploy Dai
 
 AllocatorDAOs will have Capitalization Requirements of their Effective Junior Capital based on what kind of exposure they have from their AllocatorDAO Vaults. Before the launch of SubDAOs, AllocatorDAO Advisory Teams can use this framework to deploy collateral that has no Effective Junior Capital Requirement.
 
-#### 7.1.1: ALM Capitalization Requirements.
+#### 7.1.1
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
#### 7.1.1: ALM Capitalization Requirements
```

File: MIP104/MIP104.md [hunk 2]
```diff
@@ -210,71 +232,131 @@ AllocatorDAO Advisory teams are subject to requirements for how they deploy Dai
...
+
+ALM Capitalization Requirements.
+
+##### 7.1.1.1
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
##### 7.1.1.1: ALM Tier 1 Collateral
```

File: MIP104/MIP104.md [hunk 3]
```diff
@@ -210,71 +232,131 @@ AllocatorDAO Advisory teams are subject to requirements for how they deploy Dai
...
+
+###### 7.1.1.1.3
+
+ALM Tier 1 Collateral or higher can be used for 100% of the AllocatorDAOs collateral portfolio with no Effective Junior Capital requirement.
+
+###### 7.1.1.1.3
+
+There are no additional Effective Junior Capital Requirements.
+
+##### 7.1.1.2
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
##### 7.1.1.2: ALM Tier 2 Collateral
```

File: MIP104/MIP104.md [hunk 4]
```diff
@@ -210,71 +232,131 @@ AllocatorDAO Advisory teams are subject to requirements for how they deploy Dai
...
+
+###### 7.1.1.2.3
+
+ALM Tier 2 Collateral or higher can be used for 75% of the AllocatorDAOs collateral portfolio with no Effective Junior Capital requirement.
+
+###### 7.1.1.2.4
+
+If the exposure limit is exceeded, the excess ALM Tier 2 Collateral has an additional 5% Effective Junior Capital Requirement.
+
+##### 7.1.1.3
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
##### 7.1.1.3: ALM Tier 3 Collateral
```

File: MIP104/MIP104.md [hunk 5]
```diff
@@ -210,71 +232,131 @@ AllocatorDAO Advisory teams are subject to requirements for how they deploy Dai
...
+###### 7.1.1.5.3
 
-#### 7.1.2: Risk Capitalization Requirements.
+ALM Tier 5 Collateral or higher can be used for 10% of the AllocatorDAOs collateral portfolio with no Effective Junior Capital requirement.
+
+###### 7.1.1.5.4
+
+If the exposure limit is exceeded, the excess ALM Tier 5 Collateral has an additional 20% Effective Junior Capital Requirement.
+
+#### 7.1.2
```
Comment (blimpa) [inline MIP104/MIP104.md]:
```suggestion
#### 7.1.2: Risk Capitalization Requirements
```

**Review by blimpa — CHANGES_REQUESTED** [review]
Corrections are not exhaustive. Generally, some of these are titles and should be kept next to the article code (x.x.x...); also, they should be capitalized

**Comment by blimpa** [comment]
LGTM :)

**Comment by blimpa** [comment]
Check the 1 articles: remove colons and add a new line, as we discussed

**Comment by ghost** [comment]
> Check the 1 articles: remove colons and add a new line, as we discussed

@Hipogeo please address these issue raised by @blimpa so we can merge

**Review by ghost — APPROVED** [review]

**Pull request merged**